### PR TITLE
Update on the example app

### PR DESCRIPTION
**Pull request by tomcatmwi**
### Checklist

   - Issue number for this PR: #001
   - Docs included?: no
   - Test units included?: no
   - In a clean directory, `yarn install`, `yarn test` run successfully? no

### Description
The example code in the docs didn't work on Angular 10 as `auth` was always `undefined`. It took some forum digging to find the error.

### CODE REVIEW

**Comment by hakimio** [comment]
@tomcatmwi @oavila81 @jamesdaniels 
There are two PRs for the same issue:
https://github.com/angular/angularfire/pull/2577

**Comment by Splaktar** [comment]
Also there is an even earlier PR that tries to fix this doc: https://github.com/angular/angularfire/pull/2483

File: docs/auth/getting-started.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class AppComponent {
   constructor(public auth: AngularFireAuth) {
   }
   login() {
-    this.auth.signInWithPopup(new auth.GoogleAuthProvider());
+    this.auth.signInWithPopup(new firebase.auth.GoogleAuthProvider());
```
Comment (Splaktar) [inline docs/auth/getting-started.md:33]: I think that the approach in PR https://github.com/angular/angularfire/pull/2483 is better. I.e.
```ts
this.afAuth.auth.
```

**Comment by jamesdaniels** [comment]
Thanks, addressed in another PR. Docs should be fixed now.

**Pull request closed without merging**